egs/librispeech/ASR/zipformer/train.py

```diff
@@ -950,7 +943,6 @@ def compute_loss(
             spec_augment=spec_augment,
             supervision_segments=supervision_segments,
             time_warp_factor=params.spec_aug_time_warp_factor,
```

Comment:
can not find the definition of `spec_aug_time_warp_factor`

Reply:
It is defined in `zipformer/asr_datamodule.py'